docs/source/architecture/slashing.rst

```diff
@@ -65,21 +65,21 @@ Penalty Scenarios:
 
     Result:
 
-        * 1st sub stake = 500 tokens locked for 10 periods
-        * 2nd sub stake = 200 tokens for 2 periods
-        * 3rd sub stake = 100 tokens locked starting from the next period
+        * 1st sub-stake = 500 tokens locked for 10 periods
+        * 2nd sub-stake = 200 tokens for 2 periods
+        * 3rd sub-stake = 100 tokens locked starting from the next period
         * 100 tokens in an unlocked state
 
 * *Scenario 2*: Staker incurs penalty calculated to be worth **300 tokens**:
 
-    The unlocked tokens can only cover 200 tokens. In the current period, 700 tokens are locked and 800 tokens are locked for the next period. Therefore, we should reduce amount of locked tokens for the next period and leave unchanged locked amount in the current period. The 3rd sub stake suits for this purpose but it's not the shortest one. So we take the 2nd sub stake (the shortest), reduce it to 100 tokens and add new sub stake with 100 tokens which active only in the current period.
+    The unlocked tokens can only cover 200 tokens worth of the penalty. Beyond that, the staker has 700 tokens currently locked and 100 tokens that will lock in the next period, meaning 800 tokens will be locked in total. In this scenario, we should reduce amount of locked tokens for the next period and leave unchanged locked amount in the current period. The 3rd sub-stake would be suitable to be reduced except that it's not the shortest, in terms of its unlock date. Instead, the 2nd sub-stake – the shortest (2 periods until unlock) – is reduced to 100 tokens and a new sub-stake with 100 tokens is added which is only active in the current period.
```

Comment:
"Instead, the 2nd sub-stake – the shortest (2 periods until unlock) – is reduced to 100 tokens and a new sub-stake with 100 tokens is added which is only active in the current period."

Wait... why is a new sub-stake added? This leaves us with 800 tokens when there should only be 700 remaining.

Reply:
when we reduce 2nd sub-stake to 100 then we have 600 locked in current period and 700 in the next one, so to reestablish current period we create new sub-stake

Reply:
@vzotova thanks for adding the intermediate steps. Let's add a little explanation to the last diagram for Scenario 2 explaining our creation of the 4th sub-stake. You can reuse some words from my attempt (😊 ) of an explanation in https://github.com/vzotova/nucypher/pull/5.

Similar kind of thing for Scenario 3 i.e. add an explanation that the 2nd sub-stake is emptied, but gets replenished for one period. So we didn't "create" a new sub-stake, but we did reorganize tokens so that the 2nd sub-stake had tokens 1 period.